src/main/webapp/index.html

```diff
@@ -1,10 +1,17 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Duke Programming Contest</title>
-</head>
-<body>
-Duke Programming Contest
-</body>
-</html>
\ No newline at end of file
+<html>
+  <head>
```

Comment:
let's look into using google analytics to track accesses and use. I'll make an issue separately.